integration-tests/python-enclave/host/src/test/kotlin/com/r3/conclave/integrationtests/python/enclave/PythonEnclaveTest.kt

```diff
@@ -17,7 +17,7 @@ class PythonEnclaveTest {
         fun check() {
             gramineOnlyTest()
         }
-    }
+    }c
```

Comment:
delete

Reply:
SOrry, this was to debug an integration test. Deleted now